server/server/views/profile.py

```diff
@@ -0,0 +1,49 @@
+import json
+import transaction
+
+from pyramid.config import Configurator
+from pyramid.response import Response
+from pyramid.view import view_config
+
+from server.models import get_dbsession
+
+from server.models import Base
+from server.models.user import User
+from server.models.user import users_to_json
+from server.models.user import datetime_conv, AlchemyEncoder
+
+@view_config(route_name='users.view', renderer='json')
+def get_users(request):
+    qry = get_dbsession().query(User).all()
```

Comment:
Better to separate logic and make all connections to db in models. You can create a method that will gets request like parameter for this.

Reply:
Also there already is a `dbsession`  attribute in `request` object, which automatically committed after request end.